core/core.mk

```diff
@@ -49,10 +49,21 @@ endif
 ifneq ($(CFG_TEE_CORE_DEBUG),y)
 cppflags$(sm)  += -DNDEBUG
 endif
+
+# Hardening configuration
```

Comment:
You're only referring to the comment, yes, I agree. Will fix, btw this actually challenged my git skills. The comment is already in the correct spot, so I simply cannot add a fixup, since there is nothing to change/commit. As I see my only option is to edit it in-place when doing a `git rebase -i` (which I cannot do until everything has been reviewed). If you or anyone else have something up your sleeve, please let me know.

Reply:
;-) indeed this can only be "fixed" during rebase I suppose.

Reply:
You can add two fixups. One to remove comment and other one - add it back, but to the right commit.

Reply:
Ah, brilliant!

Reply:
Fixed on patch just pushed.